libnmstate/nm/team.py

```diff
@@ -96,11 +96,9 @@ def get_info(device):
 
 def _convert_teamd_config_to_nmstate_config(team_config):
```

Comment:
I do not understand the commit message. What does `the team may return` mean - do you mean teamd?

Reply:
Yes, teamd.

Reply:
I updated the commit message.